tests/run-tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# Deploy the custom resource definitions
+kubectl create -f deploy/crds/charts.helm.k8s.io_skydives_crd.yaml
+kubectl create -f deploy/crds/charts.helm.k8s.io_netflowcollectors_crd.yaml
+
+# Install the operator
+kubectl create -f deploy/service_account.yaml
+kubectl create -f deploy/role.yaml
+kubectl create -f deploy/role_binding.yaml
+kubectl create -f deploy/operator.yaml
+
+# Create a general Skydive resource
+kubectl create -f deploy/crds/charts.helm.k8s.io_v1alpha1_skydive_cr.yaml
+# Create Skydive as netflow collector resource
+kubectl create -f deploy/crds/charts.helm.k8s.io_v1alpha1_netflowcollector_cr.yaml
+
+# Do some tests...
```

Comment:
1. For kubectl create -f deploy/crds/charts.helm.k8s.io_v1alpha1_netflowcollector_cr.yaml
kubectl get pods
We expect the same as with helm deployment - skydive pods are up and running
NAME                                                  READY   STATUS    RESTARTS   AGE
netflow-collector-skydive-agent-f8zgq                 1/1     Running   0          3m8s
netflow-collector-skydive-agent-klbmb                 1/1     Running   0          3m8s
netflow-collector-skydive-agent-zmb5m                 1/1     Running   0          3m8s
netflow-collector-skydive-analyzer-866d9778c4-bw4pk   3/3     Running   0          3m8s
skydive-operator-67c958f454-fkg9j                     1/1     Running   0          3m28s

2. The same as above for kubectl create -f deploy/crds/charts.helm.k8s.io_v1alpha1_netflowcollector_cr.yaml (after removing the above by running kubectl delete -f deploy/crds/charts.helm.k8s.io_v1alpha1_skydive_cr.yaml)

3. When deploying the charts.helm.k8s.io_v1alpha1_netflowcollector_cr.yaml we can optionally go deeper:
kubectl describe pods netflow-collector-skydive-analyzer-866
and see it has 3 containers: skydive-analyzer, skydive-exporter and skydive-minio, and all are running